tests/unit/ssg-module/test_build_yaml.py

```diff
@@ -154,7 +154,7 @@ def test_platform_from_text_simple(product_cpes):
     platform = ssg.build_yaml.Platform.from_text("machine", product_cpes)
     platform_el = platform.to_xml_element()
     assert platform_el.tag == "{%s}platform" % cpe_language_namespace
-    assert platform_el.get("id") == "machine"
+    assert platform_el.get("id") == '50f29e3d-05ac-5d11-be18-a001abdef4c3'  # "machine"
```

Comment:
This test is useless, as any modification to the ID generator will break it, and I hope that there will be modifications in the future, because these ID suck.